process/rewardTransaction/process_test.go

```diff
@@ -0,0 +1,287 @@
+package rewardTransaction_test
+
+import (
+	"errors"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/rewardTx"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	"github.com/ElrondNetwork/elrond-go/process/rewardTransaction"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewRewardTxProcessorNilAccountsDbShouldErr(t *testing.T) {
```

Comment:
where is _ in test name ?
maybe you should rename with "TestNewRewardTxProcessor_NilAccountsDbShouldErr".
If you change name you should do that for all tests from this file.

Reply:
done